e2e/imgbuild/imgbuild.go

```diff
@@ -146,15 +145,15 @@ func buildLocalImage(t *testing.T) {
 }
 
 func badPath(t *testing.T) {
-	test.EnsurePrivilege(t)
-
-	imagePath := path.Join(testenv.TestDir, "container")
-	defer os.RemoveAll(imagePath)
-
-	if b, err := e2e.ImageBuild(testenv.CmdPath, e2e.BuildOpts{}, imagePath, "/some/dumb/path"); err == nil {
-		t.Log(string(b))
-		t.Fatal("unexpected success")
-	}
+	e2e.Privileged(func(t *testing.T) {
+		imagePath := path.Join(testenv.TestDir, "container")
+		defer os.RemoveAll(imagePath)
+
+		if b, err := e2e.ImageBuild(testenv.CmdPath, e2e.BuildOpts{}, imagePath, "/some/dumb/path"); err == nil {
```

Comment:
maybe declare in a test data structure or in some global string variable to declare this path (`some/dumb/path`) instead of sending it hardcoded here

Reply:
In this PR I was forced to touch code in those files to adapt with the new privilege workflow, I'm not the original author of this code :)